modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -281,13 +282,12 @@ private JSONArray _getChildKBArticlesJSONArray(KBArticle parentKBArticle)
 			QueryUtil.ALL_POS, QueryUtil.ALL_POS, WorkflowConstants.STATUS_ANY,
 			new KBArticleTitleComparator(true));
 
-		long itemToMoveId = -1;
-		if(!ParamUtil.getString(_httpServletRequest, "itemToMoveId").isEmpty()){
-			itemToMoveId = parseLong(ParamUtil.getString(_httpServletRequest, "itemToMoveId"));
-		}
+		long kbEntryToMoveId =
+			ParamUtil.getLong(_httpServletRequest, "itemToMoveId", -1);
```

Comment:
We'll need to change this to 'kbObject' instead to 'kbEntry'. I had to make similar changes the other day by request of Brian.